samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -117,7 +122,7 @@ private enum SupportedFileType
             return this.BadRequest(ex.Message);
         }
 
-        return this.Ok();
+        return this.Accepted();
```

Comment:
Should probably not use Accepted() since Accepted means the operation may not have been completed yet.